### PR TITLE
Fix issue about UX.

### PR DESCRIPTION
**Pull request by wknet123**
Related issues are #2099, #2063, #2035, #2022, #2021, #2005, #1991, #1619, #2121

### CODE REVIEW

File: src/ui_ng/src/app/project/create-project/create-project.component.html [hunk 1]
```diff
@@ -4,10 +4,10 @@ <h3 class="modal-title">{{'PROJECT.NEW_PROJECT' | translate}}</h3>
   <div class="modal-body" style="height: 12.8em; overflow-y: hidden;">
     <form #projectForm="ngForm">
       <section class="form-block">
-        <div class="form-group">
-          <label for="create_project_name" class="col-md-4 form-group-label-override">{{'PROJECT.NAME' | translate}}</label>
+        <div class="form-group" style="padding-left: 135px;">
+          <label for="create_project_name" class="col-md-3 form-group-label-override">{{'PROJECT.NAME' | translate}}</label>
           <label for="create_project_name" aria-haspopup="true" role="tooltip" [class.invalid]="projectName.invalid && (projectName.dirty || projectName.touched)" [class.valid]="projectName.valid" class="tooltip tooltip-validation tooltip-sm tooltip-bottom-left">
```
Comment (steven-zou) [inline src/ui_ng/src/app/project/create-project/create-project.component.html]: No 'valid' class

File: src/ui_ng/src/app/log/audit-log.component.html [hunk 2]
```diff
@@ -14,12 +14,28 @@
...
+            <ng-template [ngIf]="!f.checked"><span style="display: inline-block;width: 16px;"></span></ng-template>
+            {{f.description | translate}}
+          </a> 
         </div>
       </clr-dropdown>
       <div class="flex-xs-middle">
-      <clr-icon shape="date"></clr-icon><input type="date" #fromTime  (change)="doSearchByTimeRange(fromTime.value, 'begin')">
-      <clr-icon shape="date"></clr-icon><input type="date" #toTime  (change)="doSearchByTimeRange(toTime.value, 'end')">
+          <clr-icon shape="date"></clr-icon>
+          <label for="fromDateInput" aria-haspopup="true" role="tooltip" [class.invalid]="fromTime.errors && fromTime.errors.dateValidator && (fromTime.dirty || fromTime.touched)" [class.valid]="fromTime.valid" class="tooltip tooltip-validation invalid tooltip-sm">
```
Comment (steven-zou) [inline src/ui_ng/src/app/log/audit-log.component.html]: Unreadable, should move it into a single property function in ts file.

File: src/ui_ng/src/app/replication/replication.component.html [hunk 3]
```diff
@@ -1,58 +1,62 @@
...
-                    <a href="javascript:void(0)" clrDropdownItem *ngFor="let j of jobStatus" (click)="doFilterJobStatus(j.key)"> {{j.description | translate}}</a>
-                </div>
-            </clr-dropdown>
-            <div class="flex-items-xs-middle">
-                <clr-icon shape="date"></clr-icon><input type="date" #fromTime (change)="doJobSearchByStartTime(fromTime.value)">
-                <clr-icon shape="date"></clr-icon><input type="date" #toTime (change)="doJobSearchByEndTime(toTime.value)">
-            </div>
+        <div class="flex-items-xs-middle">
+          <clr-icon shape="date"></clr-icon>
+          <label for="fromDateInput" aria-haspopup="true" role="tooltip" [class.invalid]="fromTime.errors && fromTime.errors.dateValidator && (fromTime.dirty || fromTime.touched)" [class.valid]="fromTime.valid" class="tooltip tooltip-validation invalid tooltip-sm">
```
Comment (steven-zou) [inline src/ui_ng/src/app/replication/replication.component.html]: Suggest to implement it as component

File: src/ui_ng/src/app/shared/confirmation-dialog/confirmation-dialog.component.html [hunk 4]
```diff
@@ -6,8 +6,21 @@ <h3 class="modal-title" class="confirmation-title" style="margin-top: 0px;">{{di
         </div>
         <div class="confirmation-content">{{dialogContent}}</div>
     </div>
-    <div class="modal-footer">
-        <button type="button" class="btn btn-outline" *ngIf="!confirmOnly" (click)="cancel()">{{'BUTTON.NO' | translate}}</button>
-        <button type="button" class="btn btn-primary" (click)="confirm()">{{ buttonKey | translate}}</button>
+    <div class="modal-footer" [ngSwitch]="buttons">
```
Comment (steven-zou) [inline src/ui_ng/src/app/shared/confirmation-dialog/confirmation-dialog.component.html]: Suggest to change 'caption' to support multiple button group.

File: src/ui_ng/src/app/project/create-project/create-project.component.html (hunk 1 above)
Comment (wknet123) [inline src/ui_ng/src/app/project/create-project/create-project.component.html]: Will remove this attribute.

File: src/ui_ng/src/app/shared/confirmation-dialog/confirmation-dialog.component.html (hunk 4 above)
Comment (wknet123) [inline src/ui_ng/src/app/shared/confirmation-dialog/confirmation-dialog.component.html]: Keep case-form in order to clarify multi compositions.

**Pull request merged**